plugins/file_lumberdash/lib/src/file_lumberdash.dart

```diff
@@ -0,0 +1,74 @@
+import 'dart:io';
+
+import 'package:lumberdash/lumberdash.dart';
+import 'package:synchronized/synchronized.dart';
+import 'package:meta/meta.dart';
+
+/// [LumberdashClient] that writes your logs to the given file path
+/// in the file system
+class FileLumberdash extends LumberdashClient {
+  final String _filePath;
+  File _logFile;
+  static final _lock = Lock();
+
+  FileLumberdash({
+    @required String filePath,
+  })  : assert(filePath != null),
+        _filePath = filePath;
```

Comment:
```suggestion
        _logFile = File(filePath);
```

Reply:
Ok to be out of the try/catch?

Reply:
Yeah since this is just creating a reference to the file I don't think there's any risk of it throwing an exception 👍

Reply:
Ah, sweet. Done. :)